modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,49 @@
+:experimental:
+= Migrating from CoreOS Container Linux (CL) to Fedora CoreOS (FCOS)
+
+Because Container Linux is being folded into the FCOS project, it will reach its End-of-Life sometime in 2020. This page documents differences between CL and FCOS to ease the transition to FCOS.
+
+:toc:
+
+== Introduction
+
+To migrate from CL to FCOS, you must convert your old Container Linux configs, Ignition configs, and cloud-configs to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration options have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
+
+== Installation changes
+* To accommodate changes to the Ignition specification, the `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`].
+* The autologin kernel command-line parameter is not currently supported in FCOS. To preview FCOS, you can use xref:bare-metal.adoc[Live PXE].
+* Certain CL image types are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
+* There are no official Vagrant cloud images for FCOS, and the team has no plans to create OCI container images for FCOS. FCOS is intended to be a host OS for running containers, rather than a host for producing containers.
+
+== Configuration changes
+The xref:fcct-config.adoc[Ignition specification] has changed in both syntax and function. As a result, consider the following when you write new FCC files:
+
+* Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
+//* CLC --> FCC changes. Need more info.
+//* coreos-metadata rename - need more info.
+//* PXE custom OEMs - need more info.
```

Comment:
Refers to https://coreos.com/os/docs/latest/booting-with-pxe.html#adding-a-custom-oem.  This still works with FCOS live PXE, but the Ignition config should just be `config.ign` in the root of the archive.